CHANGELOG.md

```diff
@@ -17,14 +17,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed Sequence from Wallet class
 - Core keypairs generate seed must take in hexstring instead of bytestring
 - Core keypairs formatting for ED25519 is now padded with zeros if length of keystring is less than 64
+- Removed deprecated request wrappers (the preferred method is to directly do client.request instead)
 
 ## [[Unreleased]]
 ### Added:
 - Added `submit_and_wait` to sign (if needed), autofill, submit a transaction and wait for its final outcome
 - `submit` and `send_reliable_submission` now accept an optional boolean param `fail_hard` (if `True` halt the submission if it's not immediately validated)
 
+### Changed:
+- Allowed keypairs.sign to take a hex string in addition to bytes
+
 ### Fixed:
 - Refactored `does_account_exist` and `get_balance` to avoid deprecated methods and use `ledger_index` parameter
+- Fixed crashes in the SignerListSet validation
```

Comment:
Were these in previous PRs?

Reply:
Ah, good catch - yeah I think those were from previous PRs.